catalyst-gateway/bin/src/service/api/documents/put_document/mod.rs

```diff
@@ -118,15 +119,7 @@ pub(crate) async fn endpoint(doc_bytes: Vec<u8>, mut token: CatalystRBACTokenV1)
         .into();
     }
 
-    // check if the incoming doc and the current latest doc are the same ver/id
-    let (Ok(doc_id), Ok(doc_ver)) = (doc.doc_id(), doc.doc_ver()) else {
-        return Responses::UnprocessableContent(Json(PutDocumentUnprocessableContent::new(
-            "Invalid Catalyst Signed Document",
-            serde_json::to_value(doc.problem_report()).ok(),
-        )))
-        .into();
-    };
-    match validate_against_original_doc(doc_id, doc_ver, &doc.kids()).await {
+    match validate_against_original_doc(&doc).await {
```

Comment:
Remove version from the comment to avoid confusion, also in `validate_against_original_doc` too